---
layout: post
---
# Potential for agronomic improvement in orphan crops
*News and Views article written about<sup>[1]</sup>*

---

### Of orphan crops and groundcherries

**What are orphan crops?**
<div style="text-align: justify">  
While gene editing technologies have been used to greatly improve staple crops such as corn, wheat, and rice<sup>[2]</sup>, there exist minor, regionally important but globally unknown crops whose potential for genetic improvement has not yet been fully explored. These “orphan crops” suffer from a range of agronomic issues that make them unsuitable for large scale production<sup>[3]</sup>.

This paper showcases the improvements made to the agronomic potential of the orphan crop – _Physalis pruinosa_  <span style="color:grey; font-style:italic; font-size:15px;">(groundcherry)</span> – via the genomic editing technology, CRISPR Cas-9 <span style="color:grey; font-style:italic; font-size:15px;">((Clustered regularly interspaced short palindromic repeats) (CRISPR associated protein-9 nuclease) CC9)</span>. Specifically targeting its sprawling growth habit and very strong stem abscission via orthologs <span style="color:grey; font-style:italic; font-size:15px;">(genes that are same but are present in different species)</span> of domestication genes in tomatoes.
</div>

**Why groundcherry?**
<div style="text-align: justify">
Groundcherry was favoured due to the fact it falls under Solanaceae]<sup>[1]</sup>. This family of plants contains well researched and well-known staple crops such as potatoes, tomatoes, and peppers<sup>[4]</sup>. The groundcherry is closer to tomatoes than any of the aforementioned model crops<sup>[4]</sup>, this permits the targeting of tomato domestication and improvement orthologs that are present in the groundcherry genome.
</div>
<div style="page-break-after: always;"></div>

### Techniques, Complications and Findings

**CC9 can be utilized in groundcherry**
<div style="text-align: justify">
The technical barrier to fully utilizing CC9 is the lack of efficient methodologies<sup>[5]</sup>. This can be overcome by using a technique modelled after those methodologies utilized in tomato<sup>[1]</sup>. Success was evaluated by targeting the ortholog of Sl-AG07, Ppr-AG07. Plants carrying the mutated Ppr-AG07 gene displayed the phenotype of narrower petals and leaves, thus proving that CC9 editing succeeded<sup>[1]</sup>.
</div>

**Generation of Illumina whole genome (IWG) and RNA sequencing de nova assembly (RNAsda)**
<div style="text-align: justify">
A major problem that future researchers trying to improve orphan crops can face is the lack of a reference genome alongside the absence of genomic resources. This would mean that many orthologs of domestication and improvement genes would be missing<sup>[1]</sup>. This paper demonstrates that by generating an IWG <span style="color:grey; font-style:italic; font-size:15px;">(a comprehensive method for analysing an entire genome)</span> and RNAsda <span style="color:grey; font-style:italic; font-size:15px;">(a method of creating a transcriptome without a reference genome)</span>, you can overcome such difficulties.
</div>

##### *Inducing mutations in genes controlling flowering*
<div style="text-align: justify">
Using the information provided by IWG and RNAsda, the investigators then selected the anti-florigen <span style="color:grey; font-style:italic; font-size:15px;">(anti-flowering hormone)</span>: Ppr-SP. However, targeting and nullifying Ppr-SP resulted in a severe phenotype that could limit fruit production<sup>[1]</sup>. Because of this, another anti-florigen that is also responsible for day light sensitivity, Ppr-SP5G was selected. Mutations in Ppr-SP5G eliminated day light sensitivity<sup>[1]</sup>, which resulted in up to 50% higher concentration of fruits along the shoots<sup>[1]</sup>. This has the added effect of displaying how flowering regulators can influence growth patterns.
</div>

##### *Inducing mutations in genes responsible for fruit-size*
<div style="text-align: justify">
Again, using evolutionary analysis, the researchers found an ortholog of Ppr-CLV1 in the groundcherry genome. Null mutations <span style="color:grey; font-style:italic; font-size:15px;">(mutations that result in loss of function of that gene)</span> were induced in Ppr-CLV1 that resulted in 24% increase of mass in groundcherry fruits<sup>[1]</sup>.
</div>
<div style="page-break-after: always;"></div>

### Conclusion

##### *The impact of this paper*
<div style="text-align: justify">
This study demonstrates the speed at which orphan crops can be improved and, quite possibly, be elevated to mainstream consumption and production. The gene editing technologies we have today are exceptional compared to those available a few decades ago and continue to develop at a remarkable rate<sup>[6]</sup>. Consequently, their use should not just be limited to model crops.
</div>

##### *Discussion and comments*
<div style="text-align: justify">
There are a few issues with the study. Primarily, the importance of orphan crops is assumed throughout the paper. It does do not consider that the reason orphan crops are so genetically under-developed may be the fact that they are unable to adapt to the changing climate conditions<sup>[7]</sup>. Therefore, this field might benefit from research that investigates potential orthologs for plant adaptation and sustainability instead of plant domestication and improvement. However, the attempt at improving the use of orphan crops provides grounds for further, more specified research. This report is a proof of concept of the potential for “genetic uplifting” present in orphan crops.
</div>
<div style="page-break-after: always;"></div>

### References

1. Lemmon Z, Reem N, Dalrymple J, Soyk S, Swartwood K, Rodriguez-Leal D et al. Rapid improvement of domestication traits in an orphan crop by genome editing. Nature Plants. 2018;4(10):766-770.

2. Scheben A, Wolter F, Batley J, Puchta H, Edwards D. Towards CRISPR/Cas crops - bringing together genomics and genome editing. New Phytologist. 2017;216(3):682-698.

3. Bohra A, Jha U, Kishor P, Pandey S, Singh N. Genomics and molecular breeding in lesser explored pulse crops: Current trends and future opportunities. Biotechnology Advances. 2014;32(8):1410-1428.

4. Särkinen T, Bohs L, Olmstead R, Knapp S. A phylogenetic framework for evolutionary study of the nightshades (Solanaceae): a dated 1000-tip tree. BMC Evolutionary Biology. 2013;13(1):214.

5. Van Eck J. Genome editing and plant transformation of solanaceous food crops. Current Opinion in Biotechnology. 2018;49:35-41.

6. Cermak T, Curtin S, Gil-Humanes J, Čegan R, Kono T, Konečná E et al. A multi-purpose toolkit to enable advanced genome engineering in plants. The Plant Cell. 2017;29(1196–1217):tpc.00922.2016.

7. Tadele Z. Orphan crops: their importance and the urgency of improvement. Planta. 2019;250(3):677-694.